content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -750,11 +763,16 @@ serves the following additional paths at its HTTP(S) ports.
   The output is similar to this:
 
   ```none
-  PriorityLevelName, FlowSchemaName, QueueIndex, RequestIndexInQueue, FlowDistingsher,       ArriveTime,                     UserName,              Verb,   APIPath,                                                     Namespace, Name,   APIVersion, Resource, SubResource,
-  system,            system-nodes,   12,         0,                   system:node:127.0.0.1, 2020-07-23T15:31:03.583823404Z, system:node:127.0.0.1, create, /api/v1/namespaces/scaletest/configmaps,
-  system,            system-nodes,   12,         1,                   system:node:127.0.0.1, 2020-07-23T15:31:03.594555947Z, system:node:127.0.0.1, create, /api/v1/namespaces/scaletest/configmaps,
+  PriorityLevelName, FlowSchemaName,   QueueIndex, RequestIndexInQueue, FlowDistingsher,                        ArriveTime,                     InitialSeats, FinalSeats, AdditionalLatency, UserName,                               Verb,   APIPath,                                   Namespace,   Name,   APIVersion, Resource,   SubResource
+  workload-low,      service-accounts, 14,         0,                   system:serviceaccount:default:loadtest, 2023-07-18T00:13:08.986534842Z, 10,           0,          0s,                system:serviceaccount:default:loadtest, list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
+  workload-low,      service-accounts, 14,         1,                   system:serviceaccount:default:loadtest, 2023-07-18T00:13:09.086476021Z, 10,           0,          0s,                system:serviceaccount:default:loadtest, list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
   ```
 
+  Explanation for selected column names:
+  - `InitialSeats`: The number of seats will be occupied during the initial (normal) stage of execution of the request.
+  - `FinalSeats`: The number of seats will be occupied during the final stage of the execution (accounting for the associated WATCH notifications) of the request.
+  - `AdditionalLatency`: The additional duration of the final stage of execution of the request. FinalSeats will be occupied during this time period. It does not mean any latency that a user will observe.
```

Comment:
```suggestion
  - `AdditionalLatency`: The extra time taken during the final stage of request execution.
    FinalSeats will be occupied during this time period. It does not mean any latency that a user will observe.
```

Reply:
This looks like a good improvement to me.

Reply:
Done